test/docker/grafana/add_prometheus_datasource.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
We can just put `sh` instead of `bash` if there is nothing `bash` specific in the script.

Reply:
Would be good to add this directory as a parameter to `shellcheck` in the `lint` make target.